kits/daisy/src/robot/hexapod_parameters.cpp

```diff
@@ -29,10 +46,30 @@ bool HexapodParameters::loadFromFile(std::string file)
   if (!root)
     return false;
 
+  auto body = root.child("body");
   auto stance = root.child("stance");
   auto step_thresh = root.child("step_threshold");
   auto logging = root.child("logging");
-  bool success =
+  bool success = true;
+
+  auto legs = body.children();
+
+  int leg_index = 0;
+  for (auto leg : body.children("leg")) {
+    if (leg_index > 5)
+      return false;
+    success = success &&
+      xml::trySetFloatParameter(leg.attribute("angle"), leg_angle_[leg_index]) &&
+      xml::trySetFloatParameter(leg.attribute("offset"), leg_offset_[leg_index]) &&
+      xml::trySetBoolParameter(leg.attribute("flip"), leg_flip_[leg_index]);
+    ++leg_index;
+  }
+  if (leg_index != 6)
+    return false;
+
+  success =
```

Comment:
I think we would want to && with the prior success value here?

Reply:
good catch! fixed.